torch/csrc/jit/script/init.cpp

```diff
@@ -240,8 +240,8 @@ struct VISIBILITY_HIDDEN ConstantPythonTupleValue : public PythonValue {
 
 // Represents all the parameters of a module as a List[Tensor]
 struct VISIBILITY_HIDDEN ConstantParameterList : public SugaredValue {
-  ConstantParameterList(std::shared_ptr<Module> module)
-      : module_(std::move(module)) {}
+  ConstantParameterList(std::shared_ptr<Module> module, py::list params)
+      : module_(std::move(module)), params_(std::move(params)) {}
```

Comment:
Maybe convert & store the  params to std::string here ? i imagine that's less overhead

Reply:
It'd be more since it would duplicate the strings, the `py::list` is just a thin wrapper around a Python list with the same reference semantics